uid-pages/layout-bonita/src/assets/json/localization.json

```diff
@@ -53,20 +53,18 @@
     "Display words Count": "Afficher le nombre de mots",
     "Display characters Count": "Afficher le nombre de caractères",
     "The requested content cannot be loaded.<br />Please try again later.": "Le contenu demandé ne peut pas être téléchargé.<br/> Veuillez réessayer plus tard.",
-    "User details": "",
-    "Username": "",
-    "Language": "",
-    "Select a language": "",
-    "Select a profile": "",
```

Comment:
"Select a profile" disappeared... same thing in Spanish. But it is still required in the page